server/src/stateManagement.mjs

```diff
@@ -179,38 +178,38 @@ function getUserId(userId){
   return userId;
 }
 
-// return state based on hierarchy (From lowest priority to highest) global->group->user
-function getState(userId) {
+// return state based on hierarchy (From lowest priority to highest) global->group->user if mergedState=true
+//return state of the UserId if mergedState=false,to update the state
```

Comment:
This comment seems confusing; I wouldn't expect getXxx to update. Is this just a cut-and-pasteo?